projects/core/src/i18n/translate.pipe.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { TestBed } from '@angular/core/testing';
+import { TranslationService } from './translation.service';
+import createSpy = jasmine.createSpy;
+import { Subject, of } from 'rxjs';
+import { TranslatePipe } from '.';
+import { ChangeDetectorRef } from '@angular/core';
+
+describe('TranslatePipe', () => {
+  let pipe: TranslatePipe;
+  let service: TranslationService;
+  let cd: ChangeDetectorRef;
+  let mockServiceLanguageChanged$: Subject<string>;
+
+  beforeEach(() => {
+    mockServiceLanguageChanged$ = new Subject();
+    const mockTranslateService = {
```

Comment:
Maybe `mockTranslationService` to make it consistent with the subject of mocking